.travis.yml

```diff
@@ -14,3 +14,6 @@ notifications:
   irc: "chat.freenode.net#openshift-dev"
 
 sudo: false
+
+git:
+  depth: false
```

Comment:
a few hours later . . . I figured out that


the lack of this setting was causing `/home/travis/gopath/src/github.com/openshift/origin/hack/lib/constants.sh: line 131: OS_GIT_MAJOR: unbound variable` error

Previously the depth was set to 50 and that made the build to fail. The build scripts didn't find the version using `git describe` because the history was too short and didn't have the tags for `3.9`